examples/encoding/create_simple_encoding_hls_with_multiple_audio_groups_and_stream_conditions/main.go

```diff
@@ -0,0 +1,360 @@
+		GroupID:            stringToPtr("audio_98"),
+		Language:           stringToPtr("en"),
+		AssociatedLanguage: stringToPtr("en"),
+		Name:               stringToPtr("audio_98"),
+		IsDefault:          boolToPtr(false),
+		Autoselect:         boolToPtr(false),
+		Forced:             boolToPtr(false),
+		Characteristics:    []string{"public.accessibility.describes-audio"},
+		SegmentPath:        stringToPtr("audio/98k/"),
+		EncodingID:         encodingResp.Data.Result.ID,
+		StreamID:           aacStream98kResp.Data.Result.ID,
+		MuxingID:           audioMuxing98kResp.Data.Result.ID,
+	}
+	audioMediaInfo98kResp, err := hlsService.AddMediaInfo(*hlsManifestResp.Data.Result.ID, audio98kMediaInfo)
+	errorHandler(audioMediaInfo98kResp.Status, err)
+
+	video1080pStreamInfo := &models.StreamInfo{
+		AudioGroups: &models.HLSAudioGroupConfig{
+			DroppingMode: bitmovintypes.HLSVariantStreamDroppingModeStream,
+			Groups: []models.HLSAudioGroupDefinition{
+				{
+					Name:     "audio_98",
+					Priority: 1,
+				},
+				{
+					Name:     "audio_128",
+					Priority: 10,
+				},
+			},
+		},
+		SegmentPath: stringToPtr("video/1080p/"),
+		URI:         stringToPtr("video_hi.m3u8"),
+		EncodingID:  encodingResp.Data.Result.ID,
+		StreamID:    videoStream1080pResp.Data.Result.ID,
+		MuxingID:    videoMuxing1080pResp.Data.Result.ID,
+	}
+	video1080pStreamInfoResponse, err := hlsService.AddStreamInfo(*hlsManifestResp.Data.Result.ID, video1080pStreamInfo)
+	errorHandler(video1080pStreamInfoResponse.Status, err)
+
+	video720pStreamInfo := &models.StreamInfo{
+		AudioGroups: &models.HLSAudioGroupConfig{
+			DroppingMode: bitmovintypes.HLSVariantStreamDroppingModeStream,
+			Groups: []models.HLSAudioGroupDefinition{
+				{
+					Name:     "audio_98",
+					Priority: 1,
+				},
+				{
+					Name:     "audio_128",
+					Priority: 10,
+				},
+			},
+		},
+		SegmentPath: stringToPtr("video/720p/"),
+		URI:         stringToPtr("video_lo.m3u8"),
+		EncodingID:  encodingResp.Data.Result.ID,
+		StreamID:    videoStream720pResp.Data.Result.ID,
+		MuxingID:    videoMuxing720pResp.Data.Result.ID,
+	}
+	video720pStreamInfoResponse, err := hlsService.AddStreamInfo(*hlsManifestResp.Data.Result.ID, video720pStreamInfo)
+	errorHandler(video720pStreamInfoResponse.Status, err)
+
+	// Start encoding with manifests for preview and vod
+	vodHls := models.VodHlsManifest{
+		ManifestID: *hlsManifestResp.Data.Result.ID,
+	}
+
+	options := &models.StartOptions{
+		VodHlsManifests: []models.VodHlsManifest{vodHls},
+	}
+
+	startResp, err := encodingS.StartWithOptions(*encodingResp.Data.Result.ID, options)
+	errorHandler(startResp.Status, err)
+
+	var status string
+	status = ""
+	for status != "FINISHED" {
+		time.Sleep(10 * time.Second)
+		statusResp, err := encodingS.RetrieveStatus(*encodingResp.Data.Result.ID)
+		if err != nil {
+			fmt.Println("error in Encoding Status")
+			fmt.Println(err)
+			return
+		}
+		// Polling and Printing out the response
+		fmt.Printf("%+v\n", statusResp)
```

Comment:
Maybe find a way to print better information here. This doesn't really show useful information. Is there a way to pretty print the object?

Reply:
You can use a helper method like [here](https://github.com/bitmovin/bitmovin-go/blob/master/examples/infrastructure/create_aws_infrastructure.go#L78)